src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -34,141 +34,108 @@ kubectl config current-context
 
-    The `nri-bundle` chart pulls the following dependencies, if the corresponding flags are enabled:
+Our chart has a comprehensive set of flags and tunables that can be edited to configure our solution to your particular needs. For a full list of all the flags that can be configured, please check [the chart's README](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) and the default [`values.yaml`](https://github.com/newrelic/helm-charts/blob/master/charts/nri-bundle/values.yaml) file.
 
-    | Subchart | Flag |
-    |----------|------|
-    | `newrelic-infrastructure` | `infrastructure.enabled` |
-    | `nri-prometheus` | `prometheus.enabled` |
-    | `nri-metadata-injection` | `webhook.enabled` |
-    | `kube-state-metrics` | `ksm.enabled` |
-    | `nri-kube-events` | `kubeEvents.enabled` |
-    | `newrelic-logging` | `logging.enabled` |
+4. Install the Kubernetes integration by running the customized command without `--debug` and `--dry-run`:
 
-    #### Configuring subcharts
+```
+helm upgrade --install <var>newrelic</var> newrelic/nri-bundle \
+--namespace <mark>newrelic</mark> --create-namespace \
+--set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
+--set global.cluster=<var>K8S_CLUSTER_NAME</var> \
+--set ksm.enabled=<mark>true</mark> \
+--set newrelic-infrastructure.privileged=<mark>true</mark> \
+--set infrastructure.enabled=true \
+--set prometheus.enabled=true \
+--set webhook.enabled=true \
+--set kubeEvents.enabled=true \
+--set logging.enabled=true
+```
 
-    The `nri-bundle` chart leverages [Helm's subchart configuration
-    mechanism](https://helm.sh/docs/chart_template_guide/subcharts_and_globals) to configure its components. This means
-    that you can supply the configuration values to the individual components of the chart, as listed at the beginning
-    of this section, by prefixing them with the name of the chart.
+5. Wait a few seconds, then check that the `DaemonSet` and pods have been created:
 
-    For example, to run the agent in
-    unprivileged mode, you can add `--set newrelic-infrastructure.privileged=false` to the command above, where
-    `newrelic-infrastructure` is the name of the sub-chart (as listed at the beginning of the section), and `privileged`
-    is the configuration value for said chart.
+```
+kubectl -n <mark>newrelic</mark> get daemonsets,pods
+```
 
-    Please refer to [each of the chart's README](https://github.com/newrelic/helm-charts/tree/master/charts/) and their
-    `values.yml` to see all available configuration options and their defaults.
+6. Make sure you see a `DaemonSet`, and one pod per node for `newrelic-infrastructure`, as well as one `Deployment`
+and one pod for any other component that you have enabled.
 
-  </Collapser>
+### Helm 2 using nri-bundle (recommended)
```

Comment:
I would mark it as deprecated and not as recommended

Reply:
Woops, this was certainly a copy-paste artifact. Let me fix that.